frontend/packages/ceph-storage-plugin/src/constants/ocs-install.ts

```diff
@@ -45,3 +47,43 @@ export const NetworkTypeLabels = {
   [NetworkType.DEFAULT]: 'Default (SDN)',
   [NetworkType.MULTUS]: 'Custom (Multus)',
 };
+export const KMSProviders = [
+  {
+    name: 'Vault',
+    value: 'vault',
+  },
+];
+
+export const kmsMaxFileUploadSize = 4000000;
+export const kmsFileSizeErrorMsg = 'Maximum file size exceeded. File limit is 4MB.';
+export const KMSConfigMapName = 'ocs-kms-connection-details';
+export const KMSSecretName = 'ocs-kms-token';
```

Comment:
Lets be consistent with names.
```suggestion
export const KMSMaxFileUploadSize = 4000000;
export const KMSFileSizeErrorMsg = 'Maximum file size exceeded. File limit is 4MB.';
export const KMSConfigMapName = 'ocs-kms-connection-details';
export const KMSSecretName = 'ocs-kms-token';
```